### PR TITLE
Add desynth for Muscle Belt 

### PR DESCRIPTION
**Pull request by HaxPrime**
I added muscle belt desynth recipe. I got my information from https://ffxiclopedia.fandom.com/wiki/Muscle_Belt

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4741,6 +4741,7 @@ INSERT INTO `synth_recipes` VALUES (75519,1,0,0,91,0,0,0,0,0,0,4100,4242,18406,0
 INSERT INTO `synth_recipes` VALUES (75520,1,0,0,0,85,0,0,0,0,0,4100,4242,13414,0,0,0,0,0,0,0,813,813,746,746,1,1,1,2,'Heavens_Earring_Desynth');
 INSERT INTO `synth_recipes` VALUES (75521,1,0,0,0,85,0,0,0,0,0,4100,4242,14719,0,0,0,0,0,0,0,813,813,746,746,1,1,1,2,'Heavens_Earring_+1_Desynth');
 INSERT INTO `synth_recipes` VALUES (75522,0,0,70,0,0,0,0,0,0,0,4102,4244,9091,0,0,0,0,0,0,0,19320,19320,19320,19320,1,1,1,1,'Lu Shang\'s Fishing Rod +1'); -- BGWiki and JP Wiki
+INSERT INTO `synth_recipes` VALUES (75523,1,0,0,0,92,0,0,0,0,0,4100,4242,13185,0,0,0,0,0,0,0,650,650,1117,745,1,1,1,1,'Muscle_Belt_Desynth');  -- ffxiclopedia
```
Comment (zach2good) [inline sql/synth_recipes.sql]: Double whitespace here before comment

**Review by zach2good — COMMENTED** [review]
Just ffxiclopedia tends not to be a reliable source (Insert "shit wiki says" meme). Luckily, all the other easy-to-see sources back up this level:

https://www.bg-wiki.com/ffxi/Muscle_Belt
https://ffxi.allakhazam.com/dyn/guilds/dGoldsmithing.html
https://ffxi.allakhazam.com/db/item.html?fitem=3939#Desynthesis
http://ffxi.somepage.com/itemdb/1871

and privately, Eden also has this same desynth

**Review by zach2good — APPROVED** [review]

**Comment by TeoTwawki** [comment]
http://wiki.ffo.jp/ tend to be most reliable for recipe levels

**Comment by zach2good** [comment]
Can we extend https://github.com/LandSandBoat/server/wiki/Proof-of-Work to list all the places where we might get _useful_ crafting information? And add a link to that doc to the Contributing Guide, and then yell from the rooftops to ask people to read it.

**Pull request merged**